README.md

```diff
@@ -79,6 +88,7 @@ The following  NICs were tested with this implementation. However, other SR-IOV
 - Mellanox ConnectX-5® Ex
 - Mellanox ConnectX-6®
 - Mellanox ConnectX-6® Dx
+- Mellanox BlueField-2®
```

Comment:
can you add a section about creating SFs in the Main README ?

something similar to what we have below (L95)

Reply:
L95 leed to `docs/vf-setup.md`. Should I point to chapter at `docs/subfunctions/README.md`?

Reply:
yes